### PR TITLE
Updating Sound Method

### PR DESCRIPTION
**Pull request by LoboMetalurgico**

### CODE REVIEW

File: src/main/java/cn/nukkit/level/Level.java [hunk 1]
```diff
@@ -493,15 +493,24 @@ public void addLevelEvent(Vector3 pos, int event) {
...
     }
 
+    @PowerNukkitOnly
+    @Deprecated
+    @DeprecationDetails(reason = "Old method, use addSound(pos, Sound.<SOUND_VALUE>).", since = "1.4.0.0-PN")
     public void addLevelSoundEvent(Vector3 pos, int type, int data, int entityType) {
         addLevelSoundEvent(pos, type, data, entityType, false, false);
     }
 
+    @PowerNukkitOnly
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/level/Level.java:503]: Should be @PowerNukkitDifference

File: src/main/java/cn/nukkit/level/Level.java [hunk 2]
```diff
@@ -493,15 +493,24 @@ public void addLevelEvent(Vector3 pos, int event) {
...
+    @Deprecated
+    @DeprecationDetails(reason = "Old method, use addSound(pos, Sound.<SOUND_VALUE>).", since = "1.4.0.0-PN")
     public void addLevelSoundEvent(Vector3 pos, int type, int data, int entityType, boolean isBaby, boolean isGlobal) {
         String identifier = AddEntityPacket.LEGACY_IDS.getOrDefault(entityType, ":");
         addLevelSoundEvent(pos, type, data, identifier, isBaby, isGlobal);
     }
 
+    @PowerNukkitOnly
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/level/Level.java:511]: Should be @PowerNukkitDifference

File: src/main/java/cn/nukkit/level/Level.java [hunk 3]
```diff
@@ -513,10 +522,16 @@ public void addLevelSoundEvent(Vector3 pos, int type) {
      * @param type ID of the sound from {@link cn.nukkit.network.protocol.LevelSoundEventPacket}
      * @param data generic data that can affect sound
      */
+    @PowerNukkitOnly
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/level/Level.java:525]: Should be @PowerNukkitDifference

File: src/main/java/cn/nukkit/level/Level.java [hunk 4]
```diff
@@ -513,10 +522,16 @@ public void addLevelSoundEvent(Vector3 pos, int type) {
...
+    @Deprecated
+    @DeprecationDetails(reason = "Old method, use addSound(pos, Sound.<SOUND_VALUE>).", since = "1.4.0.0-PN")
     public void addLevelSoundEvent(Vector3 pos, int type, int data) {
         this.addLevelSoundEvent(pos, type, data, ":", false, false);
     }
 
+    @PowerNukkitOnly
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/level/Level.java:532]: Should be @PowerNukkitDifference

File: src/main/java/cn/nukkit/level/Level.java [hunk 5]
```diff
@@ -913,6 +928,9 @@ public void doTick(int currentTick) {
         this.timings.doTick.stopTiming();
     }
 
+    @PowerNukkitOnly
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/level/Level.java:931]: Should be @PowerNukkitDifference

File: src/main/java/cn/nukkit/level/Level.java [hunk 6]
```diff
@@ -3747,6 +3765,9 @@ public int getUpdateLCG() {
         return (this.updateLCG = (this.updateLCG * 3) ^ LCG_CONSTANT);
     }
 
+    @PowerNukkitOnly
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/level/Level.java:3768]: Should be @PowerNukkitDifference

File: src/main/java/cn/nukkit/network/protocol/LevelSoundEventPacket.java [hunk 7]
```diff
@@ -1,8 +1,14 @@
 package cn.nukkit.network.protocol;
 
+import cn.nukkit.api.DeprecationDetails;
+import cn.nukkit.api.PowerNukkitOnly;
+import cn.nukkit.api.Since;
 import cn.nukkit.math.Vector3f;
 import lombok.ToString;
 
+@PowerNukkitOnly
```
Comment (Gabriel8579) [inline src/main/java/cn/nukkit/network/protocol/LevelSoundEventPacket.java:9]: Should be @PowerNukkitDifference

**Comment by joserobjr** [comment]
Merging, thank you for the PR 😄

**Pull request merged**